RELEASE.md

```diff
@@ -1,3 +1,45 @@
+# Release 1.12.0
+
+## Major Features and Improvements
+* Keras models can now be directly exported to the SavedModel format(`tf.contrib.saved_model.save_keras_model()`) and used with Tensorflow Serving.
+* Keras model now supports evaluating with a `tf.data.Dataset`.
+* TensorFlow binaries are built with XLA support linked in by default.
+
+## Bug Fixes and Other Changes
+
+* tf.data:
```

Comment:
We should also add a bullet for the new `tf.data.Dataset.reduce()` and `tf.data.Dataset.window()` transformations:

```
  * New `tf.data.Dataset.reduce()` API allows users to reduce a finite dataset to a single element using a user-provided reduce function.
  * New `tf.data.Dataset.window()` API allows users to create finite windows of input dataset; when combined with the `tf.data.Dataset.reduce()` API, this allows users to implement customized batching.
```